Synthesis failed for java-bigquery

Hello! Autosynth couldn't regenerate java-bigquery. :broken_heart:

Here's the output from running `synth.py`:

```
mit, working tree clean
2020-07-08 15:18:34,647 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-08 15:18:34,717 autosynth [ERROR] > Synthesis failed
2020-07-08 15:18:34,718 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0540e4 deps: update dependency com.google.apis:google-api-services-bigquery to v2-rev20200625-1.30.9 (#517)
2020-07-08 15:18:34,724 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-08 15:18:34,730 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-08 15:18:34,858 autosynth [DEBUG] > Running: git checkout a0540e4792818853a502bf56b363b26a60c5f5d9
Note: checking out 'a0540e4792818853a502bf56b363b26a60c5f5d9'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a0540e4 deps: update dependency com.google.apis:google-api-services-bigquery to v2-rev20200625-1.30.9 (#517)
2020-07-08 15:18:34,865 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-08 15:18:34,875 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-08 15:18:34,879 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-08 15:18:34,885 autosynth [INFO] > Running synthtool
2020-07-08 15:18:34,885 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-08 15:18:34,885 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-08 15:18:34,887 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-08 15:18:35,121 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-08 15:18:35,382 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-08 15:18:35,434 autosynth [ERROR] > Synthesis failed
2020-07-08 15:18:35,434 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0540e4 deps: update dependency com.google.apis:google-api-services-bigquery to v2-rev20200625-1.30.9 (#517)
2020-07-08 15:18:35,441 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-08 15:18:35,447 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/111bd893-9b98-4365-a5b5-e42fe9101d49/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
mit, working tree clean
2020-07-09 02:14:35,897 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-09 02:14:35,932 autosynth [ERROR] > Synthesis failed
2020-07-09 02:14:35,933 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0540e4 deps: update dependency com.google.apis:google-api-services-bigquery to v2-rev20200625-1.30.9 (#517)
2020-07-09 02:14:35,938 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-09 02:14:35,943 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-09 02:14:36,075 autosynth [DEBUG] > Running: git checkout a0540e4792818853a502bf56b363b26a60c5f5d9
Note: checking out 'a0540e4792818853a502bf56b363b26a60c5f5d9'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a0540e4 deps: update dependency com.google.apis:google-api-services-bigquery to v2-rev20200625-1.30.9 (#517)
2020-07-09 02:14:36,081 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-09 02:14:36,090 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-09 02:14:36,093 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-09 02:14:36,098 autosynth [INFO] > Running synthtool
2020-07-09 02:14:36,098 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-09 02:14:36,098 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-09 02:14:36,100 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-09 02:14:36,314 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-09 02:14:36,576 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-09 02:14:36,609 autosynth [ERROR] > Synthesis failed
2020-07-09 02:14:36,609 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0540e4 deps: update dependency com.google.apis:google-api-services-bigquery to v2-rev20200625-1.30.9 (#517)
2020-07-09 02:14:36,615 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-09 02:14:36,621 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/df846015-07d7-4030-b0f2-61e361036c5c/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
anch autosynth-self-1
nothing to commit, working tree clean
2020-07-09 15:13:50,502 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-09 15:13:50,534 autosynth [ERROR] > Synthesis failed
2020-07-09 15:13:50,534 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at f77e74e chore(deps): update dependency com.google.cloud:google-cloud-bigquery to v1.116.4 (#520)
2020-07-09 15:13:50,540 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-09 15:13:50,545 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-09 15:13:50,681 autosynth [DEBUG] > Running: git checkout f77e74eb3bc65a6f860677e23151bff8db1e9157
Note: checking out 'f77e74eb3bc65a6f860677e23151bff8db1e9157'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at f77e74e chore(deps): update dependency com.google.cloud:google-cloud-bigquery to v1.116.4 (#520)
2020-07-09 15:13:50,685 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-09 15:13:50,694 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-09 15:13:50,697 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-09 15:13:50,701 autosynth [INFO] > Running synthtool
2020-07-09 15:13:50,701 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-09 15:13:50,702 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-09 15:13:50,703 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-09 15:13:50,888 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-09 15:13:51,112 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-09 15:13:51,144 autosynth [ERROR] > Synthesis failed
2020-07-09 15:13:51,144 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at f77e74e chore(deps): update dependency com.google.cloud:google-cloud-bigquery to v1.116.4 (#520)
2020-07-09 15:13:51,150 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-09 15:13:51,155 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e9c50d75-a1ba-4ad3-a81e-eb3c5e7845ef/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
utosynth-self-1
nothing to commit, working tree clean
2020-07-10 02:10:45,545 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-10 02:10:45,583 autosynth [ERROR] > Synthesis failed
2020-07-10 02:10:45,583 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 82341cb deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.8.3 (#523)
2020-07-10 02:10:45,589 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-10 02:10:45,595 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-10 02:10:45,753 autosynth [DEBUG] > Running: git checkout 82341cbb12ad6f5e020b5ba15701f66f0cbbfa19
Note: checking out '82341cbb12ad6f5e020b5ba15701f66f0cbbfa19'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 82341cb deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.8.3 (#523)
2020-07-10 02:10:45,759 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-10 02:10:45,768 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-10 02:10:45,773 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-10 02:10:45,778 autosynth [INFO] > Running synthtool
2020-07-10 02:10:45,778 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-10 02:10:45,778 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-10 02:10:45,780 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-10 02:10:45,976 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-10 02:10:46,208 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-10 02:10:46,247 autosynth [ERROR] > Synthesis failed
2020-07-10 02:10:46,248 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 82341cb deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.8.3 (#523)
2020-07-10 02:10:46,254 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-10 02:10:46,259 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/bcc99092-d90a-433b-82b1-e465dc37bcbb/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-10 15:17:49,902 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-10 15:17:49,944 autosynth [ERROR] > Synthesis failed
2020-07-10 15:17:49,944 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 9b2aeb4 deps: update dependency com.google.cloud:google-cloud-storage to v1.111.2 (#531)
2020-07-10 15:17:49,951 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-10 15:17:49,957 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-10 15:17:50,098 autosynth [DEBUG] > Running: git checkout 9b2aeb4d729b0719e19fcc054eabb08527dd4ac2
Note: checking out '9b2aeb4d729b0719e19fcc054eabb08527dd4ac2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 9b2aeb4 deps: update dependency com.google.cloud:google-cloud-storage to v1.111.2 (#531)
2020-07-10 15:17:50,105 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-10 15:17:50,115 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-10 15:17:50,119 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-10 15:17:50,125 autosynth [INFO] > Running synthtool
2020-07-10 15:17:50,125 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-10 15:17:50,125 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-10 15:17:50,128 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-10 15:17:50,332 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-10 15:17:50,594 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-10 15:17:50,638 autosynth [ERROR] > Synthesis failed
2020-07-10 15:17:50,638 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 9b2aeb4 deps: update dependency com.google.cloud:google-cloud-storage to v1.111.2 (#531)
2020-07-10 15:17:50,645 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-10 15:17:50,651 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/25c51ebe-37c2-4262-ba95-6cb692394ebd/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
anch autosynth-self-1
nothing to commit, working tree clean
2020-07-11 02:11:43,548 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-11 02:11:43,591 autosynth [ERROR] > Synthesis failed
2020-07-11 02:11:43,591 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8771aff chore(deps): update dependency com.google.cloud:google-cloud-bigquery to v1.116.6 (#535)
2020-07-11 02:11:43,598 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-11 02:11:43,604 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-11 02:11:43,732 autosynth [DEBUG] > Running: git checkout 8771aff78a1359f0cb054c07214c5a273fa2c768
Note: checking out '8771aff78a1359f0cb054c07214c5a273fa2c768'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 8771aff chore(deps): update dependency com.google.cloud:google-cloud-bigquery to v1.116.6 (#535)
2020-07-11 02:11:43,738 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-11 02:11:43,748 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-11 02:11:43,751 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-11 02:11:43,757 autosynth [INFO] > Running synthtool
2020-07-11 02:11:43,757 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-11 02:11:43,757 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-11 02:11:43,759 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-11 02:11:43,965 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-11 02:11:44,222 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-11 02:11:44,258 autosynth [ERROR] > Synthesis failed
2020-07-11 02:11:44,259 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8771aff chore(deps): update dependency com.google.cloud:google-cloud-bigquery to v1.116.6 (#535)
2020-07-11 02:11:44,265 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-11 02:11:44,271 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/fb90c3c3-24b3-4b18-be76-f8d1dcf25c59/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
etadata synth.metadata synth.py --
2020-07-11 15:15:11,479 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-11 15:15:11,728 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-11 15:15:11,763 autosynth [ERROR] > Synthesis failed
2020-07-11 15:15:11,763 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-11 15:15:11,769 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-11 15:15:11,774 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-11 15:15:11,925 autosynth [DEBUG] > Running: git checkout 5cee131bd77c4ce10a579537a29f4d4cba8ddbf5
Note: checking out '5cee131bd77c4ce10a579537a29f4d4cba8ddbf5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-11 15:15:11,931 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-11 15:15:11,939 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-11 15:15:11,942 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-11 15:15:11,947 autosynth [INFO] > Running synthtool
2020-07-11 15:15:11,947 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-11 15:15:11,947 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-11 15:15:11,949 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-11 15:15:12,162 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-11 15:15:12,412 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-11 15:15:12,447 autosynth [ERROR] > Synthesis failed
2020-07-11 15:15:12,447 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-11 15:15:12,453 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-11 15:15:12,458 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/fcbc4554-5ed9-49d8-b1dd-c6899fe83fb5/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
etadata synth.metadata synth.py --
2020-07-12 02:11:52,950 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-12 02:11:53,191 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-12 02:11:53,235 autosynth [ERROR] > Synthesis failed
2020-07-12 02:11:53,235 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-12 02:11:53,241 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-12 02:11:53,247 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-12 02:11:53,388 autosynth [DEBUG] > Running: git checkout 5cee131bd77c4ce10a579537a29f4d4cba8ddbf5
Note: checking out '5cee131bd77c4ce10a579537a29f4d4cba8ddbf5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-12 02:11:53,394 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-12 02:11:53,403 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-12 02:11:53,408 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-12 02:11:53,413 autosynth [INFO] > Running synthtool
2020-07-12 02:11:53,413 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-12 02:11:53,413 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-12 02:11:53,416 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-12 02:11:53,612 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-12 02:11:53,846 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-12 02:11:53,886 autosynth [ERROR] > Synthesis failed
2020-07-12 02:11:53,887 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-12 02:11:53,893 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-12 02:11:53,898 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a2c2a1dc-063e-4381-b60e-7d2c43ba83f3/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
etadata synth.metadata synth.py --
2020-07-12 15:10:23,070 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-12 15:10:23,316 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-12 15:10:23,352 autosynth [ERROR] > Synthesis failed
2020-07-12 15:10:23,353 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-12 15:10:23,359 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-12 15:10:23,364 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-12 15:10:23,518 autosynth [DEBUG] > Running: git checkout 5cee131bd77c4ce10a579537a29f4d4cba8ddbf5
Note: checking out '5cee131bd77c4ce10a579537a29f4d4cba8ddbf5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-12 15:10:23,525 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-12 15:10:23,534 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-12 15:10:23,537 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-12 15:10:23,542 autosynth [INFO] > Running synthtool
2020-07-12 15:10:23,542 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-12 15:10:23,542 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-12 15:10:23,544 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-12 15:10:23,743 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-12 15:10:23,980 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-12 15:10:24,014 autosynth [ERROR] > Synthesis failed
2020-07-12 15:10:24,014 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-12 15:10:24,021 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-12 15:10:24,026 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9e4c4eb8-b380-4d55-a807-85094dca5983/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
etadata synth.metadata synth.py --
2020-07-13 02:16:03,144 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-13 02:16:03,412 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-13 02:16:03,453 autosynth [ERROR] > Synthesis failed
2020-07-13 02:16:03,454 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-13 02:16:03,460 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-13 02:16:03,465 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-13 02:16:03,598 autosynth [DEBUG] > Running: git checkout 5cee131bd77c4ce10a579537a29f4d4cba8ddbf5
Note: checking out '5cee131bd77c4ce10a579537a29f4d4cba8ddbf5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-13 02:16:03,604 autosynth [DEBUG] > Running: git checkout 799d8e6522c1ef7cb55a70d9ea0b15e045c3d00b
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at 799d8e6 fix: typeo in nodejs .gitattribute (#655)
2020-07-13 02:16:03,613 autosynth [DEBUG] > Running: git branch -f autosynth-12
2020-07-13 02:16:03,617 autosynth [DEBUG] > Running: git checkout autosynth-12
Switched to branch 'autosynth-12'
2020-07-13 02:16:03,622 autosynth [INFO] > Running synthtool
2020-07-13 02:16:03,622 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-13 02:16:03,622 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/12/sponge_log.log
2020-07-13 02:16:03,624 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-13 02:16:03,839 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-12
nothing to commit, working tree clean
2020-07-13 02:16:04,079 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-13 02:16:04,113 autosynth [ERROR] > Synthesis failed
2020-07-13 02:16:04,113 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5cee131 chore: release 1.116.8-SNAPSHOT (#536)
2020-07-13 02:16:04,119 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-13 02:16:04,125 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/11cd6c7d-d98e-4335-9995-b3c8394f8e4a/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
ynth.py --
2020-07-13 15:16:22,140 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-13 15:16:22,401 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-13 15:16:22,461 autosynth [ERROR] > Synthesis failed
2020-07-13 15:16:22,461 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a6d9491 docs(samples): add get table labels (#540)
2020-07-13 15:16:22,469 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-13 15:16:22,476 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-13 15:16:22,630 autosynth [DEBUG] > Running: git checkout a6d94913f74a6b4d29e9314a0652a9e3fa30aed9
Note: checking out 'a6d94913f74a6b4d29e9314a0652a9e3fa30aed9'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a6d9491 docs(samples): add get table labels (#540)
2020-07-13 15:16:22,638 autosynth [DEBUG] > Running: git checkout d82deccf657a66e31bd5da9efdb96c6fa322fc7e
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at d82decc build: missing closing paren in publish script (#659)
2020-07-13 15:16:22,649 autosynth [DEBUG] > Running: git branch -f autosynth-13
2020-07-13 15:16:22,653 autosynth [DEBUG] > Running: git checkout autosynth-13
Switched to branch 'autosynth-13'
2020-07-13 15:16:22,659 autosynth [INFO] > Running synthtool
2020-07-13 15:16:22,659 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-13 15:16:22,659 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/13/sponge_log.log
2020-07-13 15:16:22,662 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-13 15:16:22,882 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-13
nothing to commit, working tree clean
2020-07-13 15:16:23,149 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-13 15:16:23,204 autosynth [ERROR] > Synthesis failed
2020-07-13 15:16:23,204 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a6d9491 docs(samples): add get table labels (#540)
2020-07-13 15:16:23,215 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-13 15:16:23,222 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/bbb9fc42-c3ea-45c7-a24b-c571860301d6/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

Autosynth is still having trouble generating java-bigquery. :sob:

Here's the output from running `synth.py`:

```
ynth.py --
2020-07-14 02:14:19,108 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-07-14 02:14:19,364 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-14 02:14:19,428 autosynth [ERROR] > Synthesis failed
2020-07-14 02:14:19,428 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a6d9491 docs(samples): add get table labels (#540)
2020-07-14 02:14:19,435 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-07-14 02:14:19,441 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-14 02:14:19,575 autosynth [DEBUG] > Running: git checkout a6d94913f74a6b4d29e9314a0652a9e3fa30aed9
Note: checking out 'a6d94913f74a6b4d29e9314a0652a9e3fa30aed9'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a6d9491 docs(samples): add get table labels (#540)
2020-07-14 02:14:19,581 autosynth [DEBUG] > Running: git checkout d82deccf657a66e31bd5da9efdb96c6fa322fc7e
Previous HEAD position was 4f2c9f7 ci(java): run dependency test on Java 8 and 11 (#639)
HEAD is now at d82decc build: missing closing paren in publish script (#659)
2020-07-14 02:14:19,591 autosynth [DEBUG] > Running: git branch -f autosynth-13
2020-07-14 02:14:19,595 autosynth [DEBUG] > Running: git checkout autosynth-13
Switched to branch 'autosynth-13'
2020-07-14 02:14:19,601 autosynth [INFO] > Running synthtool
2020-07-14 02:14:19,601 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-14 02:14:19,601 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-bigquery/13/sponge_log.log
2020-07-14 02:14:19,603 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-14 02:14:19,805 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-bigquery/synth.py.
On branch autosynth-13
nothing to commit, working tree clean
2020-07-14 02:14:20,050 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-bigquery/synth.py", line 27, in <module>
    '.kokoro/presubmit/java11-samples.cfg',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 365, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 121, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 105, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 49, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-07-14 02:14:20,101 autosynth [ERROR] > Synthesis failed
2020-07-14 02:14:20,101 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a6d9491 docs(samples): add get table labels (#540)
2020-07-14 02:14:20,108 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-14 02:14:20,114 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/744c1ebc-59ec-439b-ae0c-6d79d561de7c/targets/github%2Fsynthtool;config=default/tests;query=java-bigquery;failed=false).

--- a/samples/snippets/src/main/java/com/example/bigquery/DeleteLabelTable.java
+++ b/samples/snippets/src/main/java/com/example/bigquery/DeleteLabelTable.java
@@ -54,4 +54,4 @@ public class DeleteLabelTable {
     }
   }
 }
-// [START bigquery_delete_label_table]
+// [END bigquery_delete_label_table]

